src/midgard/util.cc

```diff
@@ -89,6 +89,45 @@ template <class container_t> container_t trim_front(container_t& pts, const floa
   return result;
 }
 
+void TrimShape(float start,
```

Comment:
minor nit: camel-cased `TrimShape` looks inconsistent with the rest of the convention used in this file, which is snake-case. Maybe rename this to `trim_shape`?

Reply:
Fixed, although I'll note that we don't have consistent naming conventions within the larger codebase.